components/filter/filter.js

```diff
@@ -266,17 +266,32 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 				</d2l-dropdown-menu>
 			`;
 
+		const countBadgeTemplate = filterCount ? html`
+				<d2l-count-badge
+					type="count"
+					max-digits=5
```

Comment:
The [default max-digits for d2l-count-badge](https://github.com/BrightspaceUI/core/blob/main/components/count-badge/count-badge-mixin.js#L12) is currently 5. We discussed it very briefly but it wasn't super clear whether she was ok with the default (which I mentioned was 5) vs really wanting 5 regardless of the default.

Reply:
Yah wasn't sure if either solution here mattered too much. Feels like we can just let it follow whatever the default for countBadge `max-digits` ends up being?
I'll remove this explicit set then